db/schema.rb

```diff
@@ -10,18 +10,31 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_09_17_084624) do
+ActiveRecord::Schema.define(version: 2018_09_29_123828) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "addresses", force: :cascade do |t|
```

Comment:
Is this related to AP-133?
Adding client address.
Should be included with this PR.

Reply:
Seems like this needs to be ✂️ as it is not part of this PR like Charlie mentioned.